.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: groovy
+language: java
```

Comment:
Language is Groovy for Grails a project. https://guides.grails.org/grails-on-travis-basics/guide/index.html

Reply:
Switched